src/pages/404.js

```diff
@@ -83,7 +83,7 @@ const NotFoundPage = () => {
 
   return !loading ? (
     <Layout className="landing">
-      <Seo title="404: Not found" />
+      <Seo title="404: Not found | OurPetPolicy" description="404: Not Found" />
```

Comment:
why are meta titles and descriptions hardcoded. We should read them from admin portal. Please review example contact page. Also please apply this to all(there are more below). Thanks

Reply:
Hi, @nonahayrapetyan got it, I will do it. The site was made that way, so I didn't change anything because of that. But I will be pushing soon with updates.